client/src/CountVotes.js

```diff
@@ -0,0 +1,71 @@
+import { useState } from "react";
+import ThumbUp from "@material-ui/icons/ThumbUp";
+import ThumbDown from "@material-ui/icons/ThumbDown";
+
+const CountVotes = () => {
+  const [likeCount, setLikeCount] = useState(0);
```

Comment:
These three pieces of state all only ever change together, which means they maybe could be collapsed into one piece of state (maybe with values "like", "neutral", and "dislike") - what do you think?